samples/tutorials/hardwareInterfaces/gpio/Cf3GpioControl/cf3GpioControlComponent/cf3GpioControl.c

```diff
@@ -0,0 +1,83 @@
+//--------------------------------------------------------------------------------------------------
+/** 
+ * This sample app reads state of  IoT1_GPIO1 (gpio25)
+ * If state change is detected, it makes corresponding change in state of LED D750
+ * Use this sample to understand how to configure a gpio as an input or output
+ * and use call back function
+ */
+//--------------------------------------------------------------------------------------------------
+
+#include "legato.h"
+#include "interfaces.h"
+
+static int8_t sensorGpioValue;
+
+//--------------------------------------------------------------------------------------------------
+/**
+ * Sets default configuration LED D750 as off
+ */
+//--------------------------------------------------------------------------------------------------
+static void ConfigureLed
+(
+    void
+)
+{
+    // Configure initial value as LED off
+   
+    LE_FATAL_IF(mangoh_ledGpio_SetPushPullOutput(MANGOH_LEDGPIO_ACTIVE_HIGH, true) != LE_OK,
+                "Couldn't configure LED PLAY as a push pull output");
+}
+
+//--------------------------------------------------------------------------------------------------
+/**
+ * Performs initial configuration of the CF3 gpio (IoT1_GPIO1)
+ */
+//--------------------------------------------------------------------------------------------------
+static void ConfigureSensorGpio
+(
+    void
+)
+{
+    // Configure IoT1_GPIO1 as input and set its initial value as high
+
+    LE_FATAL_IF(le_sensorGpio_SetInput(LE_SENSORGPIO_ACTIVE_HIGH ) != LE_OK,
+                "Couldn't configure cf3 gpio as default input high");
+}
+
+
+//--------------------------------------------------------------------------------------------------
+/**
+ * LED D750 changes state when IoT1_GPIO1 changes state
+ */
+//--------------------------------------------------------------------------------------------------
+static  void touch_ledGpio_ChangeCallback
```

Comment:
`touch_ledGpio_ChangeCallback` is an odd function name.  Maybe `touchSensorChangeHandler` would be better.